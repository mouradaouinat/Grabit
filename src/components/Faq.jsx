import React from 'react'

const Faq = () => {
  return <h1 className="settings__heading">Faq</h1>
}

export default Faq
