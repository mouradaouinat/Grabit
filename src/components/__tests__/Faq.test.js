import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Faq from '../Faq'

Enzyme.configure({ adapter: new Adapter() })

describe('Faq', () => {
  it('renders without crashing', () => {
    shallow(<Faq />)
  })

  it('displays the same content every time it renders', () => {
    const snapshot = Faq
    expect(snapshot).toMatchSnapshot()
  })
})
