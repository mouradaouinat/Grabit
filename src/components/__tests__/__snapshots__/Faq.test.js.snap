// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faq displays the same content every time it renders 1`] = `[Function]`;
